trace-utils/src/send_data/mod.rs

```diff
@@ -211,7 +211,7 @@ impl SendData {
         };
 
         match Client::builder()
-            .build(connector::Connector::default())
+            .build(connector::Connector::new(true))
```

Comment:
This code is shared with the data-pipeline and sidecar trace sender.

I wonder whether it makes more sense to have this thing as a rust feature in Cargo.toml instead of constructor arguments.
At least to me it seems like this way it can be centrally enabled for the mini agent specifically, without touching individual places where the connector is used.

Also, this trivializes any concerns around binary size.

Reply:
Great suggestion! I'll look into making this a feature